includes/documents/abstract-wcpdf-order-document-methods.php

```diff
@@ -82,21 +82,23 @@ public function get_billing_address() {
 		// always prefer parent billing address for refunds
 		if ( $this->is_refund( $this->order ) ) {
 			// temporarily switch order to make all filters / order calls work correctly
-			$refund = $this->order;
+			$refund      = $this->order;
 			$this->order = $this->get_refund_parent( $this->order );
-			$address = apply_filters( 'wpo_wcpdf_billing_address', $this->order->get_formatted_billing_address(), $this );
+			$address     = $this->order->get_formatted_billing_address();
 			// switch back & unset
 			$this->order = $refund;
-			unset($refund);
-		} elseif ( $address = $this->order->get_formatted_billing_address() ) {
-			// regular shop_order
-			$address = apply_filters( 'wpo_wcpdf_billing_address', $address, $this );
+			unset( $refund );
 		} else {
+			// regular shop_order
+			$address = $this->order->get_formatted_billing_address();
+			
 			// no address
-			$address = apply_filters( 'wpo_wcpdf_billing_address', __('N/A', 'woocommerce-pdf-invoices-packing-slips' ), $this );
+			if ( empty( $address ) ) {
+				$address = __( 'N/A', 'woocommerce-pdf-invoices-packing-slips' );
+			}
```

Comment:
Shouldn't we put this outside the if/else statement, so it also works for refunds with a parent order that has no address?